tests/wp_viewporter.cpp

```diff
@@ -104,23 +108,23 @@ class WpViewporterTest : public wlcs::InProcessServer
         }
 
         // Moving any further should take us out of the surface
-        motion_received = false;
+        *motion_received = false;
 
         client.add_pointer_leave_notification(
-            [&](auto)
+            [pointer_entered](auto)
             {
-                pointer_entered = false;
-                return false;
-            });
-        client.add_pointer_motion_notification(
-            [&](auto, auto)
-            {
-                motion_received = true;
+                *pointer_entered = false;
                 return false;
             });
 
+        client.add_pointer_motion_notification([motion_received](auto, auto)
+        {
+            *motion_received = true;
+            return false;
+        });
```

Comment:
Why reformat this lambda? You didn't see a need to reformat the other three

Reply:
Forgot to format it in the rush to get CI fixed. Should be all good now.